core/integration/features/verified/0019-MCAL-margin_calculator-slippage.feature

```diff
@@ -79,6 +79,7 @@ Feature: Test capped maximum slippage values are calculated correctly in range o
         And the parties should have the following margin levels:
             | party | market id | maintenance | search | initial | release |
             | party | ETH/MAR23 | 85690       | 94259  | 102828  | 119966  |
+#margin = min((100000-15900), 15900*(100))+0.1*15900=85690
```

Comment:
This doesn't match the spec for AC [0019-MCAL-025](https://github.com/vegaprotocol/specs/blob/palazzo/protocol/0019-MCAL-margin_calculator.md#0019-MCAL-025)?

Spec expects maintenance of `1 591 590`

Reply:
spec has been updated, please review again :)